src/gui_tool/VideoFileManager.py

```diff
@@ -39,10 +52,17 @@ def get_frame_index(self) -> int:
         return int(self.capture.get(cv2.CAP_PROP_POS_FRAMES))
 
     def is_open(self) -> bool:
-        return self.capture.isOpened() and self.get_frame_index() <= self.get_frames_count()
+        return self.capture.isOpened() and self.get_frame_index() < self.get_frames_count()
```

Comment:
so I think right now the gui tool will close once we reach the end. Can we make it just pause instead of close? At least for me I always accidentally close the GUI by running the video to the end, especially when the clip is short

Reply:
Hmm... It should already
Tell me if it still happens once this gets merged in

Reply:
Does it error when it closes?

Reply:
ya it did, trying to replicate on master now but I'm getting another error: 
```
printer loaded, item <src.data.VideoItem.VideoItem object at 0x7fdfd8028090>, message: 
[ERROR:0] global /io/opencv/modules/videoio/src/cap.cpp (116) open VIDEOIO(CV_IMAGES): raised OpenCV exception:

OpenCV(4.2.0) /io/opencv/modules/videoio/src/cap_images.cpp:293: error: (-215:Assertion failed) !_filename.empty() in function 'open'


Traceback (most recent call last):
  File "./run", line 6, in <module>
    pipeline.main()
  File "/home/nick/Workspace/AutoDash/src/pipeline.py", line 39, in main
    run(config, **args)
  File "/home/nick/Workspace/AutoDash/src/pipeline.py", line 63, in run
    item = executor.run(item)
  File "/home/nick/Workspace/AutoDash/src/executor/Labeler.py", line 7, in run
    tag_file(item.filepath, item.metadata)
  File "/home/nick/Workspace/AutoDash/src/gui_tool/entry.py", line 41, in tag_file
    gui.start()
  File "/home/nick/Workspace/AutoDash/src/gui_tool/gui_managers.py", line 60, in start
    self.set_GUI()
  File "/home/nick/Workspace/AutoDash/src/gui_tool/gui_managers.py", line 87, in set_GUI
    set_progress_rate_callback)
cv2.error: OpenCV(4.2.0) /io/opencv/modules/highgui/src/window_QT.cpp:1223: error: (-27:Null pointer) Max value of the slider must be bigger than 0 in function 'addSlider2'
```